lib/src/client_impl.dart

```diff
@@ -50,6 +50,14 @@ class MeiliSearchClientImpl implements MeiliSearchClient {
     return MeiliSearchIndexImpl.fromMap(this, response.data!);
   }
 
+  @override
+  Future<Map<String, dynamic>> getRawIndex(String uid) async {
+    final response =
+        await http.getMethod<Map<String, dynamic>>('/indexes/$uid');
```

Comment:
DRY with the `getIndex` method 😇

Reply:
Good catch, I create a new method called _getIndex (a private method), and this will respond with the response from the `/indexes/$uid` request then each method will deal with that.